src/EnergyPlus/EPVector.hh

```diff
@@ -91,6 +91,7 @@ template <typename T> struct EPVector : private std::vector<T>
 
     void deallocate() noexcept
     {
+        m_allocated = false;
```

Comment:
@lefticus I encountered an issue trying to run EnergyPlus twice in a row.  In our clear state methods, we routinely ask any allocated state arrays to deallocate, or clear.  In my testing, I found that even though I was calling the method to deallocate the array, somehow the object was still showing that it was indeed allocated, and caused the second run of EnergyPlus to fail.  I found that the EPVector m_allocated member was not set back to false in a couple places.  Can you confirm this looks OK?